src/main/java/cmput402/tdd/Recipe.java

```diff
@@ -20,39 +20,39 @@ public void setName(String name){
         this.name = name;
     }
 
-    public void add(Item item, Integer quantity) throws RuntimeException{
+    public void add(Item item, Integer quantity) throws Exception{
         if(items.get(item) == null && quantity > 0){
             items.put(item, quantity);
         }
         else if(items.get(item) != null && quantity > 0){
             items.put(item, items.get(item)+quantity);
         }
         else{
-            throw new RuntimeException("Error: item cannot be added");
+            throw new Exception("item cannot be added");
         }
     }
     public void add(Item item){
         items.put(item, 1);
     }
 
-    public void remove(Item item, Integer quantity) throws RuntimeException{
+    public void remove(Item item, Integer quantity) throws Exception{
         if(items.get(item) != null && items.get(item)> quantity && quantity > 1){
             items.put(item, items.get(item)-quantity);
         }
         else if(items.get(item) != null && items.get(item) == quantity){
             items.remove(item);
         }
         else{
-            throw new RuntimeException("Error: item cannot be removed from recipe");
+            throw new Exception("item cannot be removed from recipe");
         }
     }
 
-    public void remove(Item item) throws RuntimeException{
+    public void remove(Item item) throws Exception{
```

Comment:
If this method is not being called anywhere else can it be changed to private?

Reply:
Method is currently being called in RecipeTests